addons/im_livechat/static/tests/chat_window_patch.test.js

```diff
@@ -27,5 +27,6 @@ test("closing a chat window with no message from admin side unpins it", async ()
     await click(".o-mail-ChatWindow-command[title*='Close Chat Window']", {
         parent: [".o-mail-ChatWindow", { text: "Demo" }],
     });
-    await contains(".o_notification", { text: "You unpinned your conversation with Demo" });
+    await openDiscuss();
+    await contains(".o-mail-DiscussSidebarChannel", { count: 0, text: "Demo" });
```

Comment:
This `contains()` can pass due to using an outdated classname, this isn't that reliable :/

You could add another chat in test, so you assert with similar selector but on other chat, to give a better chance the selector is relevant.